activiti-cloud-api/activiti-cloud-api-process-model/src/main/java/org/activiti/cloud/api/process/model/CloudProcessDefinition.java

```diff
@@ -16,8 +16,16 @@
 package org.activiti.cloud.api.process.model;
 
 import org.activiti.api.process.model.ProcessDefinition;
+import org.activiti.api.process.model.VariableDefinition;
 import org.activiti.cloud.api.model.shared.CloudRuntimeEntity;
 
+import java.util.List;
+
 public interface CloudProcessDefinition extends CloudRuntimeEntity, ProcessDefinition {
 
+    @Override
```

Comment:
Why do we need it?

Reply:
I've added `getVariableDefinitions()` to ProcessDefinition in activiti core, but in activiti-cloud ProcessInstanceEntity implements it (by implementing CloudProcessDefinition). This way I won't have to implement this method in entity, which would look weird.

Reply:
The implementation moved to ExtendedCloudProcessDefinition.